app/client/authzed/client.go

```diff
@@ -0,0 +1,39 @@
+package authzed
```

Comment:
Instead of this package, we should imo have another file under `host/imp/` called something like `SpiceDBAuthzStore.go`, implementing the existing AuthzStore interface defined in `app/dependencies`. This would be the only file where the authzed client is needed, integrated and called. Can we change this?

Reply:
Ok I can see now why a separate client package may be useful to only instantiate one client.

Given the current architectures, if I understand it correctly, the `app` package reflects the "domain" package in DDD. The constraint of this package is to not be tied to any technical implementation.

So I'd suggest we move the implementation, together with the actual store, in a package `host/impl/spicedb` instead of the app  package. wdyt?

(same goes for "flags" btw. Not sure about utils placement, it feels to me that utils should be a library like apache commons if it is really that much used, then it would be clear where to place it)